.travis.yml

```diff
@@ -28,6 +39,10 @@ env:
   - TRAVIS_NODE_VERSION="5" ARCH="x86"
   - TRAVIS_NODE_VERSION="6"
   - TRAVIS_NODE_VERSION="6" ARCH="x86"
+  - TRAVIS_ELECTRON_VERSION="0.37.8" ARCH="x64" TRAVIS_NODE_VERSION="5"
```

Comment:
That's the version currently running in Electron (specified in npm_config_disturl )